src/pooling_api.cpp

```diff
@@ -46,6 +46,11 @@ inline void Pooling_logging_cmd(const miopenPoolingDescriptor_t poolDesc,
         {
         case miopenHalf: ss << "poolfp16"; break;
         case miopenFloat: ss << "pool"; break;
+        case miopenInt32:
+        case miopenInt8:
+        case miopenInt8x4:
+        case miopenBFloat16:
+        case miopenDouble:
         default: MIOPEN_THROW(miopenStatusNotImplemented);
```

Comment:
Let's avoid fatal errors in the debugging code. It is the library core that should do that. Here I recommend printing things like "poolint8_UNIMPLEMENTED", "poolbfp16_UNIMPLEMENTED", "poolfp64_UNIMPLEMENTED".

Reply:
Resolved.